lib/partner/partnerFields.js

```diff
@@ -0,0 +1,62 @@
+import {
+  GraphQLString,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+} from 'graphql';
+
+import { fetchAllPartners } from '../fetch';
+
+const PartnerTypeFields = {
+  id: {
+    type: GraphQLString,
+    description: 'Unique id of partner',
+  },
+  name: {
+    type: new GraphQLNonNull(GraphQLString),
```

Comment:
Here we're saying this is not nullable, but it looks like we may return null for this field in the sanitizer bit.